### PR TITLE
!jira tips command

### LINKED ISSUE #113: !jira tips for new members
!jira tips would list tips for making a bug report or commenting/making requests, such as searching before making a report, explanation of duplicates, clones, avoid making requests when a mod is taking care of new reports etc, as it could be useful for new members.

### PR DESCRIPTION
**Pull request by chandler05**
## Purpose
Close #113
## Future work
The wording of the tips message could definitely be changed, feedback on it is probably needed

### CODE REVIEW

**Review by dericksonmark — CHANGES_REQUESTED** [review]
I've never reviewed anything before but here is one. I don't know if I needed to write the line numbers in my comments, so I did it anyway. Sorry if I did anything wrong!

File: src/commands/TipsCommand.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	public async run( message: Message ): Promise<boolean> {
+		try {
+			await message.channel.send(
+				`Welcome to the Mojira Discord Server! 
+				
+				For help using the bug tracker, there is a helpful article on the Minecraft website that you can read: https://help.minecraft.net/hc/en-us/articles/360049840492 
+				
+				How to use this server: 
+				Start by choosing which bug tracker projects you would like to be a part of in #role-selection
+				Afterwards, you can use correspondidng request channels in each project to make requests for things on the bug tracker, like resolutions and adding affected versions.
```
[the remark this answers is not in the document]
Comment (chandler05) [reply src/commands/TipsCommand.ts]: Thanks for the feedback :)

[the remark this answers is not in the document]
Comment (dericksonmark) [reply src/commands/TipsCommand.ts]: Thank you for making the changes :)

**Review by dericksonmark — APPROVED** [review]

File: src/commands/TipsCommand.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { Message } from 'discord.js';
+import PrefixCommand from './PrefixCommand';
+import BotConfig from '../BotConfig';
```
Comment (SPGoding) [inline src/commands/TipsCommand.ts:3]:
```
GitHub Actions / build (12.x)

src/commands/TipsCommand.ts#L3

'BotConfig' is defined but never used
```

File: src/commands/TipsCommand.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+export default class TipsCommand extends PrefixCommand {
+	public readonly aliases = ['tips', 'tutorial'];
+
+	public async run( message: Message ): Promise<boolean> {
+		try {
+			await message.channel.send(
+				`Welcome to the Mojira Discord Server! 
+				
+				For help using the bug tracker, there is an article on the Minecraft website that you can read: https://help.minecraft.net/hc/articles/360049840492
```
Comment (SPGoding) [inline src/commands/TipsCommand.ts:13]:
```suggestion
				For help with using the bug tracker, there is an article on the Minecraft website that you can read: <https://help.minecraft.net/hc/articles/360049840492>.
```

File: src/commands/TipsCommand.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+		try {
+			await message.channel.send(
+				`Welcome to the Mojira Discord Server! 
+				
+				For help using the bug tracker, there is an article on the Minecraft website that you can read: https://help.minecraft.net/hc/articles/360049840492 
+				
+				How to use this server: 
+				Start by choosing which bug tracker projects you would like to be a part of in #role-selection.
+				Afterwards, you can use corresponding request channels in each project to make requests for changes to tickets on the bug tracker, like resolutions and adding affected versions. 
+				The mods and helpers of the bug tracker will then be able to see the requests and resolve them.`.replace( /\t/g, '' )
```
Comment (SPGoding) [inline src/commands/TipsCommand.ts:18]:
```suggestion
				The moderators and helpers of the bug tracker will then be able to see the requests and resolve them.`.replace( /\t/g, '' )
```

**Review by SPGoding — APPROVED** [review]

**Pull request merged**